### ISSUE TITLE
Duplicate street labels

### ISSUE THREAD

**Issue by LinusGeffarth** [thepeaklab/GoogleMapsTileOverlay#9]
Hey there, I noticed that when using the map styles, duplicate street labels are added to the map, cluttering it. Compare those two screenshots:

1. without the styling
<img width="222" alt="Bildschirmfoto 2020-03-30 um 11 06 56" src="https://user-images.githubusercontent.com/9455439/77895204-b8c75500-7276-11ea-9c16-fd34d2514a1d.png">

2. with styling
<img width="244" alt="Bildschirmfoto 2020-03-30 um 11 06 20" src="https://user-images.githubusercontent.com/9455439/77895223-bebd3600-7276-11ea-9b28-a442a6b2cba0.png">

The zoom level is almost identical.

Here's my json (renamed the file to `.txt` to upload on Github, was originally `.geojson`): 
[gmapsStyles.txt](https://github.com/thepeaklab/GoogleMapsTileOverlay/files/4401785/gmapsStyles.txt)

Thanks!

**Comment by Pic2490** [comment]
@LinusGeffarth 

I've found you can turn off labels in the google maps styling wizard so that MapKit labels are the only ones shown, if you're still looking into this

**Comment by LinusGeffarth** [comment]
Good point. I didn't assume that both maps' labels are shown but now that you're saying it, it looks obvious.  
Thanks for the hint!
